gatsby/src/styles/global.scss

```diff
@@ -34,7 +34,9 @@ ul {
 	color: #44525e;
 	font-size: 18px;
 	line-height: 29px;
-  padding-bottom: 1em;
+}
+ul:not(.nav.toc-list.nav-list) {
```

Comment:
@carolyn-shannon I _think_ that the list of classes should be treated as a logical OR list, but I'm not sure.